logic/src/commonMain/kotlin/com/wire/kalium/logic/sync/receiver/MissedNotificationsEventReceiver.kt

```diff
@@ -39,6 +39,7 @@ internal class MissedNotificationsEventReceiverImpl(
 ) : MissedNotificationsEventReceiver {
 
     override suspend fun onEvent(event: Event.AsyncMissed, deliveryInfo: EventDeliveryInfo): Either<CoreFailure, Unit> {
+        slowSyncRepository.setNeedsToPersistHistoryLostMessage(true)
```

Comment:
We need to keep retrocompatibility with the old system when we were offline for too long.